### ISSUE TITLE
Brèves inaccessibles

### ISSUE THREAD

**Issue by yvandouenel** [jabberwooki/crealead#114]
les co-entrepreneurs et les référents identifiés sur le site de crealead n'ont pas le droit de voir certaines brèves :

Le contenu Appel à Projets Région sur la Création et la Reprise d'entreprise : manifestez votre intérêt (Brève) correspondant à l'une de vos alertes vient d'être créé.

**Comment by SandMat** [comment]
Non, celle-là, c'est parce que j'ai créé la brève mais que je ne l'ai pas publiée. Le vrai bug (que j'ai mis dans GitHub) c'est qu'elle n'aurait pas dû être envoyée dans l'alerte.
